src/core/sign.ml

```diff
@@ -278,8 +280,8 @@ let read : string -> t = fun fname ->
   in
   reset_timed_refs sign
 
-(* NOTE here, we rely on the fact that a marshaled closure can only be read by
-   processes running the same binary as the one that produced it. *)
+(** NOTE here, we rely on the fact that a marshaled closure can only be read
```

Comment:
keep as normal comment (does not need to appear in the ocaml doc)

Reply:
Done